infra/modular-examples/gcp-google-workspace/main.tf

```diff
@@ -13,12 +13,13 @@ locals {
 
 module "worklytics_connector_specs" {
   source = "../../modules/worklytics-connector-specs"
-  # source = "git::https://github.com/worklytics/psoxy//infra/modules/worklytics-connector-specs?ref=v0.4.22"
+  # source = "git::https://github.com/worklytics/psoxy//infra/modules/worklytics-connector-specs?ref=v0.4.22
 
   enabled_connectors             = var.enabled_connectors
   google_workspace_example_user  = var.google_workspace_example_user
-  google_workspace_example_admin = coalesce(var.google_workspace_example_admin, var.google_workspace_example_user)
+  google_workspace_example_admin = try(coalesce(var.google_workspace_example_admin, var.google_workspace_example_user), null)
   salesforce_domain              = var.salesforce_domain
+  msft_tenant_id                 = var.msft_tenant_id
```

Comment:
this is the fix, pass through `msft_tenant_id` to `worklytics_connector_specs` module

Reply:
doesn't really matter, as example doesn't create msft connectors anyways. this whole thing needs refactor at some point (to better decouple data sources from host env; and to refactor `aws` case so that google, microsoft can be split out if unused - rather than requiring gcloud, az tooling to be installed and auth'd)